shutter/low-level.md

```diff
@@ -447,54 +448,63 @@ def get_participating_validators(state) -> Sequence[ValidatorIndex]:
                 version,
                 chain_id,
                 validator_registry_address,
-                validator_index,
+                start_index,
+                count,
                 nonce,
                 is_registration,
             ) = extract_message_parts(update.message)
         except:
             continue
 
+        if count == 0:
+            continue
         if version != VALIDATOR_REGISTRY_MESSAGE_VERSION:
             continue
         if chain_id != CHAIN_ID:
             continue
         if validator_registry_address != VALIDATOR_REGISTRY_ADDRESS:
             continue
 
-        if validator_index >= len(state.validators):
-            continue
-        if nonce <= prev_nonces.get(validator_index, -1):
+        end_index = start_index + count
```

Comment:
If start and end index are inclusive, it should be `end_index = start_index + count - 1` -> If count == 1 then start_index == end_index

Reply:
I decided to make end index non-inclusive as it works better with python syntax (eg. `state.validators[start_index : end_index]` and `range(start_index, end_index)`). You think it would be clearer to make end index inclusive?